src/config/uio-assets.json

```diff
@@ -14,6 +14,7 @@
     "src/framework/enhancement/css/",
     "src/framework/core/css/",
     "src/framework/core/images/",
+    "src/lib/hypher/patterns",
```

Comment:
It doesn't look like syllabification is being used, so this isn't necessary. Although leaving it in would make it easier to configure UIO to use syllabication in the future.

Reply:
@jobara That's the idea.